src/deploy/functions/runtimes/python/index.ts

```diff
@@ -30,9 +29,15 @@ export async function tryCreateDelegate(
     logger.debug("Customer code is not Python code.");
     return;
   }
-  const runtime = context.runtime ? context.runtime : LATEST_VERSION;
-  if (!runtimes.isValidRuntime(runtime)) {
-    throw new FirebaseError(`Runtime ${runtime} is not a valid Python runtime`);
+  const runtime = context.runtime ? context.runtime : supported.latest("python");
```

Comment:
Nit: `context.runtime ?? supported.latest("python")` is probably cleaner than a ternary